extensions/stackdriver/stackdriver.cc

```diff
@@ -425,12 +425,15 @@ inline bool StackdriverRootContext::enableEdgeReporting() {
   return config_.enable_mesh_edges_reporting() && !isOutbound();
 }
 
-bool StackdriverRootContext::shouldLogThisRequest() {
+bool StackdriverRootContext::shouldLogThisRequest(
+    ::Wasm::Common::RequestInfo& request_info) {
   std::string shouldLog = "";
   if (!getValue({::Wasm::Common::kAccessLogPolicyKey}, &shouldLog)) {
     LOG_DEBUG("cannot get envoy access log info from filter state.");
     return true;
   }
+  // Add label log_sampled if Access Log Policy sampling was applied to logs.
+  request_info.log_sampled = (shouldLog != "no");
```

Comment:
super nit: could we just do string comparison for one time? Just return request_info.log_sampled in the next line.

Reply:
thanks.. done